apps/back-office/__tests__/utils/emailService.send.integration.test.ts

```
@@ -0,0 +1,204 @@
+      },
+    });
+  });
+
+  afterAll(async () => {
+    if (appointmentTemplate?.id) {
+      await prisma.emailTemplate.delete({
+        where: { id: appointmentTemplate.id },
+      });
+    }
+
+    delete process.env.GMAIL_USER;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)